src/Nirum/Constructs/Annotation/Internal.hs

```diff
@@ -20,7 +23,11 @@ import Nirum.Constructs (Construct (toCode))
 import Nirum.Constructs.Docs
 import Nirum.Constructs.Identifier (Identifier)
 
-type AnnotationArgumentSet = M.Map Identifier T.Text
+data AnnotationArgument = AText T.Text
+                        | AInt Int
```

Comment:
- [`Int`][1] has a limits on its size (up to `0x1FFFFFFF`).  We'd better to use [`Integer`][2] instead.
- Data constructors shouldn't be prefixed with `A`.

[1]: https://www.stackage.org/haddock/lts-11.8/base-4.10.1.0/Data-Int.html
[2]: https://www.stackage.org/haddock/lts-11.8/base-4.10.1.0/Prelude.html#t:Integer